opensrp-giz-malawi/build.gradle

```diff
@@ -28,11 +28,11 @@ android {
     buildToolsVersion androidBuildToolsVersion
 
     defaultConfig {
-        applicationId "org.smartregister.giz_malawi"
+        applicationId "org.smartregister.giz"
```

Comment:
This will affect the app directly. It can no longer be updated, the data cannnot be reused. The app will be installed as another app and beside the current `org.smartregister.giz_malawi` app registered. 

If the app is being used, then this should be reverted

Reply:
The release made was made with `giz` as the package name and the same is used on fabric.

Reply:
This is not going to change any other time I assume.

Reply:
Cool